modules/update-cluster-version-object.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="update-cluster-version-object_{context}"]
+= The ClusterVersion object
+
+One of the resources that the Cluster Version Operator (CVO) monitors is `ClusterVersion`.
+
+Administrators and OpenShift components can communicate or interact with the CVO through the `ClusterVersion` object.
+The desired CVO state is declared through the `ClusterVersion` object and the current CVO state is reflected in the object's status.
+
+[NOTE]
+====
+Do not directly modify the `ClusterVersion` object. Instead, use interfaces such as the `oc` CLI or the web console to declare your desired update target.
+====
+
+The CVO continually reconciles the cluster with the target state declared in `ClusterVersion` `spec`.
```

Comment:
Not sure if its suppose to be `ClusterVersion.spec`.

Reply:
So these are intentionally spaced out as separate words, but I can try and reword this to have more clarity. Maybe "the `spec` property of the `ClusterVersion` resource"

Reply:
Gotcha, I really wasn't too sure on this one but I had a thought. But that sounds good to me 👍